src/app/rpmostree-builtin-status.c

```diff
@@ -563,6 +563,12 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
           print_kv ("Unlocked", max_key_len, unlocked);
           g_print ("%s%s", bold_suffix, red_suffix);
         }
+      const char *end_of_life_string = NULL;
+      /* look for ostree.endoflife attribute in the deployment */
+      g_variant_dict_lookup (dict, "endoflife", "&s", &end_of_life_string);
+
+      if (end_of_life_string)
+       g_print ("\n end-of-life: %s", end_of_life_string);
```

Comment:
Nit: there's a space between the new line and the "end-of-life" string. Is that intended?